task.go

```diff
@@ -208,7 +208,7 @@ func (r *marathonClient) TaskEndpoints(name string, port int, healthCheck bool)
 	return list, nil
 }
 
-func (r *Task) allHealthChecksAlive() bool {
+func (r *Task) AllHealthChecksAlive() bool {
```

Comment:
The implementation assumes that we are dealing with a task configured with custom health checks. This means that users need to make sure they are calling the method on proper tasks only (like the single caller in go-marathon does) to avoid returning a negative result for an otherwise healthy task.

I see two options to mitigate this problem:

1. Document the circumstance in the method's godoc.
1. Create a new method on `Application` that returns a list of healthy tasks, taking any configured health checks into account.

I'm leaning towards 2., not sure though if that fits your use case. Any thoughts / preferences?